CHANGELOG.md

```diff
@@ -1,11 +1,9 @@
-# NEXT RELEASE
+# 12.5.1 Release notes
```

Comment:
Should the **NEXT RELEASE** block be added to this file?

Reply:
I am going to add it later (at the end of the release process via this: https://ci.realm.io/job/core_release_notes_next/)